cmd/usage.go

```diff
@@ -0,0 +1,86 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/cobra/doc"
+)
+
+func newUsageCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:    "usage [PATH] (default is \"docs\")",
+		Short:  "Generate usage documentation in markdown for the CLI.",
+		Hidden: true,
+		RunE:   usage,
+		Args:   cobra.MaximumNArgs(1),
+	}
+}
+
+var defaultDocsPath = "docs"
+
+// nolint: gosec
+func isCmdAvailable(name string) bool {
+	cmd := exec.Command("/bin/sh", "-c", "command", "-v", name)
+	if err := cmd.Run(); err != nil {
+		return false
+	}
+	return true
+}
+
+func usage(cmd *cobra.Command, args []string) error {
```

Comment:
I’d rather we did the call to Pandoc from outside go. I’d like to generate the .md content from go, and the exit. 

Then in config.yml call out to pandoc to generate HTML.